app/scripts/utils/veda-data/index.ts

```diff
@@ -0,0 +1,50 @@
+import { useParams } from 'react-router';
+import { 
+  DatasetData, StoryData, VedaData
+} from "$types/veda";
+
+// @NOTE: Independent of veda faux modules
+
+/**
+ * List with the meta information of all datasets.
+ */
+export const getAllDatasetsProps = (datasets: VedaData<DatasetData>) => Object.values(datasets).map((d) => d!.data);
+
+/**
+ * List with the meta information of all stories.
+ */
+// @QUESTION: This was migrated over from the original /scripts/utils/veda-data file but doesn't seem to be used anywhere, we should get rid of it I think
+// Also it does the same thing as getAllDatasetsProps
+export const getAllStoriesProps = (stories: VedaData<StoryData>) => Object.values(stories).map(
+  (d) => d!.data
+);
```

Comment:
We can probably get rid of this as it wasn't used in the original file either

Reply:
+1 for removing if not used